cumulusci/tasks/bulkdata/load.py

```diff
@@ -53,17 +51,13 @@ def _init_options(self, kwargs):
             self.options.get("ignore_row_errors", False)
         )
         if self.options.get("sql_path"):
-            if self.options.get("database_url"):
-                raise TaskOptionsError(
-                    "The database_url option is set dynamically with the sql_path option.  Please unset the database_url option."
-                )
             self.options["sql_path"] = os_friendly_path(self.options["sql_path"])
-            if not os.path.isfile(self.options["sql_path"]):
-                raise TaskOptionsError(
-                    f"File {self.options['sql_path']} does not exist"
-                )
```

Comment:
Removed this because `open` in "r" mode will raise a very similar exception if the file doesn't exist.